x/stakeibc/keeper/msg_server_redeem_stake.go

```diff
@@ -66,22 +67,19 @@ func (k Keeper) RedeemStake(goCtx context.Context, msg *types.MsgRedeemStake) (*
 	}
 	native_tokens := inCoin.Amount.ToDec().Mul(rate).TruncateInt()
 	outCoin := sdk.NewCoin(hostZone.HostDenom, native_tokens)
+	_ = outCoin
```

Comment:
does this line do anything? same comment for ln 75

Reply:
The unimplemented part of this function requires these coins, but the linter removes these lines unless you do this